File: 01_Programming_Basics/03_Programming_Basics_with_JavaScript/04_Nested_Conditional_Statements/04_FruitOrVegetable.js
function FruitOrVegetable(args) {
  let input = args[0].toLowerCase();
  let msg = "unknown";

  if (input == "banana" || input == "apple" || input == "kiwi" || input == "cherry" || input == "lemon" || input == "grapes") { 
    msg = "fruit"; 
  } else if (input == "tomato" || input == "cucumber" || input == "pepper" || input == "carrot") { 
    msg = "vegetable"; 
  }

  console.log(msg);

}

File: 01_Programming_Basics/01_Programming_Basics_with_C++/06_Nested_Conditional_Statements_Lab/04_Fruit_Or_Vegetable.cpp
#include <iostream>
using namespace std;

int main() {
	string product, msg;
	cin >> product;

	if (product == "banana" || product == "apple" || product == "kiwi" || product == "cherry" || product == "lemon" || product == "grapes") {
	msg = "fruit";
	} else if (product == "tomato" || product == "cucumber" || product == "pepper" || product == "carrot") {
	msg = "vegetable";
	} else {
	msg = "unknown";
	}

	cout << msg << endl;

	return 0;

}

